crates/revm/src/db/states/bundle_state.rs

```diff
@@ -8,7 +8,9 @@ use revm_interpreter::primitives::{
     AccountInfo, Bytecode, HashMap, StorageSlot, B160, B256, KECCAK_EMPTY, U256,
 };
 
-/// Bundle state contain only values that got changed
+// mattsse: It's not clear to me what exactly a bundle is, is this a collection of multiple executed transactions that are bundled together?
```

Comment:
Created issue for overview docs this here: https://github.com/bluealloy/revm/issues/683